tapchannel/aux_funding_controller.go

```diff
@@ -549,7 +549,7 @@ func newCommitBlobAndLeaves(pendingFunding *pendingAssetFunding,
 
 	// Just like above, we don't have a real HTLC view here, so we'll pass
 	// in a blank view.
-	fakeView := &lnwallet.HtlcView{}
+	fakeView := lnwallet.AuxHtlcView{}
```

Comment:
nit: since this isn't a pointer to a struct anymore, this could just be `var fakeView lnwallet.AuxHtlcView`.